package.yaml

```diff
@@ -7,7 +7,8 @@ author:
   - Philippe Laprade
   - Jean-Pierre Rupp
   - Matthew Wraith
-maintainer: jprupp@protonmail.ch
+  - Keagan McClelland
+maintainer: keagan.mcclelland@gmail.com
 license: MIT
```

Comment:
I believe we can (and should) change the license to BSD3.

Reply:
That doesn't require soliciting permission does it, since MIT is less restrictive?